assignment2/src/views/AboutView.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import Para from '../components/MainPara.vue'
+</script>
+
+<template>
+    <Para title="About" body="This is a neat little project for CSCD 379 - .NET Web Application Development. It's meant to emulate the Wordle browser game, and in some instances perhaps even exceed it in expectations. Made with Vue/Vuetify, primarily Typescript and a little CSS/HTML."></Para>
+    <v-btn to="/" prepend-icon="mdi-chevron-left" append-icon="mdi-chevron-right" block size="large" elevation="12" variant="outlined" color="primary" class="button"> Back </v-btn>
```

Comment:
I don't think this is a huge deal, but a back button typically takes you to the previous page that you visited. This button might as well say "Home" instead of "Back"

Reply:
That's a good point. Thanks.